library/core/class.controller.php

```diff
@@ -1863,6 +1863,26 @@ public function renderMaster() {
                 // This is done in the controller rather than the asset model because the translations are not linked to compiled code.
                 $this->Head->addScript(url('/api/v2/locales/'.rawurlencode(Gdn::locale()->current())."/translations?js=1&x-cache=1&h=$busta", true), 'text/javascript', false, ['defer' => 'true']);
 
+                $polyfillFileUrl = asset("/js/polyfills/core-polyfills.js", false, $addVersion);
```

Comment:
Can you make this a method in the AssetModel so this can be re-used on non Gdn_Controller classes?